cpp/util/masterelection.cc

```diff
@@ -88,6 +99,8 @@ MasterElection::MasterElection(const shared_ptr<libevent::Base>& base,
       backed_proposal_(kNoBacking),
       is_master_(false) {
   CHECK_NE(kNoBacking, node_id);
+  is_master_gauge->Set(0);
+  participating_in_election_gauge->Set(0);
```

Comment:
ok, I couldn't resist it, but I take your point that there could be other less crazy situations where you have this arise.

I guess if you mean that there are e.g. two entirely separate elections going on for some reason then I suppose giving the elections a name and having that as a label helps in that particular scenario, multiple participants in the same election in the same binary is a bit off the wall.

I suppose we'll cross this particular bridge if we come to it?